presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -152,6 +153,11 @@ protected void assertInvalidFunction(String projection, ErrorCodeSupplier expect
         functionAssertions.assertInvalidFunction(projection, expectedErrorCode);
     }
 
+    protected void assertFunctionThrowsIncorrectly(@Language("SQL") String projection, Class<? extends Throwable> throwableClass, @Language("RegExp") String message)
```

Comment:
not used?

Reply:
It will be used in other similar PRs, like #17232